### PR TITLE
[release-v1.23] Ensure we can specify different CDI CR name. (#1383)

### PR DESCRIPTION
**Pull request by awels**
Fix bug where the CR name was hard coded to CDI and it failed in HCO.

Signed-off-by: Alexander Wels <awels@redhat.com>

**What this PR does / why we need it**:
Backport of #1383 
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by awels** [comment]
needs /lgtm

**Comment by maya-r** [comment]
travis is failing

File: automation/test.sh [hunk 1]
```diff
@@ -29,6 +29,10 @@ export CDI_NAMESPACE=$NAMESPACE
 
 echo "namespace: ${NAMESPACE}, cdi-namespace: ${CDI_NAMESPACE}"
 
+if [[ -n "$RANDOM_CR" ]]; then
```
Comment (maya-r) [inline automation/test.sh:32]: does anything ever set RANDOM_CR?

Comment (awels) [reply automation/test.sh:32]: I am going to set it in project-infra as soon as I am done getting this out the door. Probably in the ceph lane.

Comment (awels) [reply automation/test.sh:32]: https://github.com/kubevirt/project-infra/pull/645

**Pull request merged**